src/helpers/helpers.curve.ts

```diff
@@ -1,11 +1,40 @@
 import {almostEquals, distanceBetweenPoints, sign} from './helpers.math';
 import {_isPointInArea} from './helpers.canvas';
+import {ChartArea} from '../../types';
+
+export interface SplinePoint {
+  x: number;
+  y: number;
+  skip?: boolean;
+
+  // Both Bezier and monotone interpolations have these fields
+  // but they are added in different spots
+  cp1x?: number;
+  cp1y?: number;
+  cp2x?: number;
+  cp2y?: number;
+}
+
+/**
+ * Keep for backwards compatibility
+ */
+export type MonotoneSplinePoint = SplinePoint
```

Comment:
If we want to squeeze this into v4, then I can remove the `MonotoneSplinePoint` type entirely otherwise this has to wait until v5

Reply:
I'd try to get these in v4

Reply:
ok, will remove `MonotoneSplinePoint` and mark this as breaking